### ISSUE TITLE
issue when runnig test_vit.sh

### ISSUE THREAD

**Issue by Blissy-32** [YvanYin/Metric3D#137]
Hello , after preparing the JSON file for my own dataset according to the instructions, is that a need to modify the _data_base_.py（
img_size=(540, 960),
focal_length=1196.0,
）to my camera，and the resolution must over 616,1024?

**Comment by JUGGHM** [comment]
> Hello , after preparing the JSON file for my own dataset according to the instructions, is that a need to modify the _data_base_.py（ img_size=(540, 960), focal_length=1196.0, ）to my camera，and the resolution must over 616,1024?

Not necessarily. We usually rewrite data_basic in the training config files to override it:
https://github.com/YvanYin/Metric3D/blob/277c3a1311da2d69816600f8e09c7b292146843b/training/mono/configs/RAFTDecoder/vit.raft5.giant2.nyu.py#L41-L51
